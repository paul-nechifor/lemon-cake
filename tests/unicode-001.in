(= ă 'litera ă mică')
ă
