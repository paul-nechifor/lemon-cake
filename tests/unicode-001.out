'litera ă mică'
'litera ă mică'
